src/index.html

```diff
@@ -7,6 +7,442 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
looks like font doesn't work and it is reason of wrong all tests
set such link for fix fonts
```suggestion
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="./styles/main.scss">
```